server/vulnerabilities/macoffice/sync.go

```diff
@@ -46,6 +46,9 @@ func sync(
 	}
 
 	if len(local) == 0 {
+		if _, err := ghClient.Download(url); err != nil {
```

Comment:
@sharon-fdm @lucasmrod There was a silly bug with this statement. If no local files were present, then no remote files were ever downloaded.